tests/functional/sign_up.js

```diff
@@ -66,7 +62,7 @@ define([
   }
 
   registerSuite({
-    name: 'sign_up',
+    name: 'sign_up - ours',
```

Comment:
Doh. I added that so I could grep="sign_up - ours" w/o running 100 extra tests. Removing, thanks for catching that!